modules/exploits/multi/http/jenkins_script_console.rb

```diff
@@ -100,10 +106,28 @@ def http_send_command(cmd, _opts = {})
         }
     }
     request_parameters['vars_post'][@crumb[:name]] = @crumb[:value] unless @crumb.nil?
-    res = send_request_cgi(request_parameters)
-    if !(res && (res.code == 200))
+    send_request_cgi(request_parameters)
+  end
+
+  # This method takes a command and options then attempts to make a request to send the command
+  #
+  # @param [String] cmd The cmd used
+  # @param [String] _opts Request options
+  # @return [Rex::Proto::Http::Response] res The response of the request
+  def http_send_command(cmd, _opts = {})
+    res = http_send_request(cmd)
+
+    if res.code == 401
```

Comment:
res could be null here, you're not checking it until `!(res && (res.code == 200))` further on down